### PR TITLE
Improve readme with more project information

### PR DESCRIPTION
**Pull request by otaviojava**
This PR aims to add more information and explain more details in the README file.

# Major Change

- Update README file

# Minor Change

### CODE REVIEW

**Review by mpredli01 — COMMENTED** [review]
Hi @otaviojava:

This is awesome!  Three minor things:
[a] I double-checked and Denver is indeed in Mountain time
[b] is there a way to implement the `Repository` interface without having to use `SeContainer` and `Service` like I do in my Jakarta NoSQL/MongoDB application?  I think this would be useful for line 29 for those that may not know how to create the instance. 
[c] you have "==" on line 29.

File: README.adoc [hunk 1]
```diff
@@ -19,7 +92,7 @@ link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org
 * America: 
 link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Toronto++[Eastern],
 link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Chicago++[Central],
-link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Denver++[Mountain],
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Denver++[Western],
```
Comment (njr-11) [inline README.adoc:95]: Leave this as `Mountain` time. `Western` will be confusing.

File: README.adoc [hunk 2]
```diff
@@ -3,10 +3,83 @@
...
+
+== Code of Conduct
+
+This project is governed by the Eclipse Foundation of Conduct. By participating, you are expected to uphold this code of conduct. Please report unacceptable behavior to codeofconduct@eclipse.org.
+
+== Getting Help
+
+Having trouble with Jakarta Data? We’d love to help!
+
+Report bugs with Jakarta Data at https://github.com/otaviojava/jakarta-data/issues.
```
Comment (njr-11) [inline README.adoc:72]:
```suggestion
Report bugs with Jakarta Data at https://github.com/jakartaee/data/issues.
```

File: README.adoc [hunk 3]
```diff
@@ -3,10 +3,83 @@
...
+=== Repository
+
+A repository abstraction aims to significantly reduce the boilerplate code required to implement data access layers for various persistence stores.
+
+[source,java]
+----
+@Repository
+public interface CarRepository extends CrudRepository<Car, Long> {
+
+  List<Car> findByType(Type type);
```
Comment (njr-11) [inline README.adoc:19]: Avoid including the`Type` class in the example if we don't define what it is.  Users will wonder if it comes from the specification and where to find it.  An easy change would be to switch it to `String`.

File: README.adoc [hunk 4]
```diff
@@ -3,10 +3,83 @@
...
+
+  Optional<Car> findByName(String name);
+
+}
+----
+
+
+[source,java]
+----
+CarRepository repository ==//instance
```
Comment (njr-11) [inline README.adoc:29]:
```suggestion
@Inject
CarRepository repository;
...
```

File: README.adoc [hunk 5]
```diff
@@ -3,10 +3,83 @@
...
+
+== Getting Help
+
+Having trouble with Jakarta Data? We’d love to help!
+
+Report bugs with Jakarta Data at https://github.com/otaviojava/jakarta-data/issues.
+
+== Building from Source
+
+You don’t need to build from source to use the project, but if you want to try, you can make it using Maven and Java 11 or higher.
```
Comment (njr-11) [inline README.adoc:76]:
```suggestion
You don’t need to build from source to use the project, but you can do so with Maven and Java 11 or higher.
```

**Review by njr-11 — CHANGES_REQUESTED** [review]

**Review by njr-11 — APPROVED** [review]

File: README.adoc [hunk 6]
```diff
@@ -3,10 +3,85 @@
...
+  <artifactId>jakarta-data-api</artifactId>
+  <groupId>jakarta.data</groupId>
+  <version>1.0.0-SNAPSHOT</version>
+</dependency>
+----
+
+
+== Code of Conduct
+
+This project is governed by the Eclipse Foundation of Conduct. By participating, you are expected to uphold this code of conduct. Please report unacceptable behavior to codeofconduct@eclipse.org.
```
Comment (pzygielo) [inline README.adoc:68]:
```
This project is governed by the Eclipse Foundation ... of Conduct. 
```
:eyes: Is something missing from that sentence?

Comment (otaviojava) [reply README.adoc]: Good, point!
Thanks

**Pull request merged**